### PR TITLE
Add settings for Share extension

### PR DESCRIPTION
**Pull request by AgentIsai**
This PR adds settings for the Share extension

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3965,12 +3965,37 @@
...
+	'wgShareUsePlainLinks' => [
+		'default' => true,
+	],
+	
+	// Site notice opt out
 	'wmgSiteNoticeOptOut' => [
 		'default' => false,
 	],
-
+
```
Comment (Universal-Omega) [inline LocalSettings.php:3998]: Tabbing

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -2429,6 +2429,78 @@
 			],
 		],
 	],
+	'wgShareUsePlainLinks' => [
+		'name' => 'Use plain links for sidebar share menu',
+		'from' => 'share',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'socialtools',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:2437]: If we are adding this to "Social tools", we need to rename the section to "Social", as "Social tools" is the social tools extensions (IE SocialProfile), or configuration involving the social tools extensions.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php (hunk 2 above)
Comment (AgentIsai) [reply ManageWikiSettings.php:2437]: If you're fine with that then sure but I added it there as I really can't think of where else to put it.

File: LocalSettings.php (hunk 1 above)
Comment (AgentIsai) [reply LocalSettings.php]: Fixed, thanks

File: ManageWikiSettings.php (hunk 2 above)
Comment (Universal-Omega) [reply ManageWikiSettings.php:2437]: I will just do the section rename later. This can be merged as-is for now I suppose.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**